server/schedule/checker/merge_checker_test.go

```diff
@@ -122,8 +122,8 @@ func (s *testMergeCheckerSuite) SetUpTest(c *C) {
 				},
 			},
 			&metapb.Peer{Id: 109, StoreId: 4},
-			core.SetApproximateSize(10),
-			core.SetApproximateKeys(10),
+			core.SetApproximateSize(1),
```

Comment:
Here, the original value 10 is greater than threshold 2, which causes the region to be unable to be merged.